### ISSUE TITLE
[no-call-expression] Add support for including/excluding specific functions

### ISSUE THREAD

**Issue by AbdealiLoKo** [angular-eslint/angular-eslint#1055]
**Description and reproduction of the issue**
I am trying to avoid using `form.get(...)` in my templates and force my team to use `form.controls.` in the HTML templates.
And I came across this rule - which is pretty awesome.

But I realized I get a lot of false positives - because some library maintainers recommend using their functions in the template:
https://ngneat.github.io/transloco/docs/translation-in-the-template#structural-directive

And they have said that the function is optimized and created to be used in templates efficiently (via memoization etc)

So, my request is - can we enable/disable certain calls based on the name of the function ?
Maybe a `includeNames`/`excludeNames` optional configuration for this rule ?

**Versions**

| package                                   | version |
| ----------------------------------        | ------- |
| `@angular-eslint/eslint-plugin-template`  | `13.0.1` |
| `@angular-eslint/template-parser`         | `13.0.1` |
| `@typescript-eslint/parser`               | `5.29.0` |
| `ESLint`                                  | `8.18.0` |
| `node`                                    | `16.13.2` |

```sh
# Please run `npx ng version` in your project and paste the full output here:

$ npx ng version

     _                      _                 ____ _     ___
    / \   _ __   __ _ _   _| | __ _ _ __     / ___| |   |_ _|
   / △ \ | '_ \ / _` | | | | |/ _` | '__|   | |   | |    | |
  / ___ \| | | | (_| | |_| | | (_| | |      | |___| |___ | |
 /_/   \_\_| |_|\__, |\__,_|_|\__,_|_|       \____|_____|___|
                |___/


Angular CLI: 14.0.0
Node: 16.13.2
Package Manager: npm 8.1.2
OS: linux x64

Angular: 14.0.0
... animations, cli, common, compiler, compiler-cli, core, forms
... language-service, platform-browser, platform-browser-dynamic
... router

Package                         Version
---------------------------------------------------------
@angular-devkit/architect       0.1400.0
@angular-devkit/build-angular   14.0.0
@angular-devkit/core            14.0.0
@angular-devkit/schematics      14.0.0
@schematics/angular             14.0.1
rxjs                            7.5.5
typescript                      4.7.3

```

- [ ] I have tried restarting my IDE and the issue persists.
- [ ] I have updated to the latest supported version of the packages and checked my `ng version` output per the instructions given here.

**Comment by theorlovsky** [comment]
would be really nice to see it in a future version
